app/views/documents/form_partials/_upload.html.erb

```diff
@@ -1,5 +1,9 @@
 <div class="govuk-form-group <%= form.object.errors[:file].any? ? 'govuk-form-group--error' : '' %>">
-  <%= form.label :file, "Upload a file", class: "govuk-label" %>
+  <% if current_page?(new_planning_application_document_path(@planning_application)) %>
```

Comment:
Small thing, but I wonder if `@document.new_record?` rather than checking the request path is a more reliable and direct way of doing this?

Reply:
Thanks, yes - have implemented this. It's better.